### ISSUE TITLE
Greek Anthology is divided in five urns

### ISSUE THREAD

**Issue by PonteIneptique** [PerseusDL/canonical-greekLit#133]
And it feels wrong, should the CapiTainS version be only one URN ? In that case, @balmas, can the catalog handle this situation ??

**Comment by balmas** [comment]
I believe the Greek anthology catalog data is wrong, and it should be a single urn. We need to invalidate the existing 5 urns and make a new one that they all redirect to.  This should be entered on catalog_data as an issue

**Comment by AlisonBabeu** [comment]
So this is a rather vexing problem. The TLG ID for the Greek Anthology is 7000.001, and is used for the entire work/edition. The Loeb edition that we used for the Perseus Digital Library was published in five volumes and was published on Perseus as five separate TEI files, each of which got their own legacy DOC ID as well as their own CTS URNs, all using the top level TLG. In order to get MODS records for these volumes into the Perseus catalog, I simply used the URNS they already had, namely 
tlg7000.tlg001.perseus-grc1, tlg7000.tlg001.perseus-grc2, tlg7000.tlg001.perseus-grc3, tlg7000.tlg001.perseus-grc4, :tlg7000.tlg001.perseus-grc5. I hadn't worried that much in that for all of the various authors in the Greek Anthology, I have always cataloged them under their individual TLGs when they had them. I know this is technically incorrect, but wasn't sure of any other way to make sure all five Perseus documents were represented in the Perseus Catalog.

There is another issue going on in that the author Damostratus, which has the TLG 7000a (I created it, don't ask), was mis-cataloged for a SLUB collection under TLG 7000, which I think has caused a problem with the Textgroup name in that the online catalog is associating this author name with the Work/Textgroup of the Greek Anthology.

I will check the CiteCollection tables and fix that issue but as to how to handle the CTS URN for the five volumes of the Greek Anthology on Perseus and the Catalog, I'm not entirely sure as to the best solution.

**Comment by PonteIneptique** [comment]
Just so I am sure for conversion : it would be ok to merge everything to grc6, because it represents the same work / same edition ? And then we would have to deal with it in Perseus Catalog :/

**Comment by AlisonBabeu** [comment]
Yes, grc6 would be o.k. because it all these files and URNs are indeed part of the same edition.
